docs/technical_specifications/PROVIDER.md

```diff
@@ -14,18 +14,27 @@ can be inherited in the general settings file of your project.
 ````
```

Comment:
we have more settings params, standing on provder.settings

please, can you document all of them here?